common/remote/CMakeLists.txt

```diff
@@ -51,16 +52,22 @@ if (USE_URIPARSER)
 endif(USE_URIPARSER)
 
 include_directories (
-         ./../../system/hrpc 
-         ./../../system/mp 
-         ./../../system/include 
-         ./../../system/jlib 
-         ./../../system/jhtree
-         ./../../rtl/eclrtl
-         ./../../system/security/securesocket
-         ./../../system/security/cryptohelper
-         ./../../testing/unittests
-         ./../../rtl/include
+         ${HPCC_SOURCE_DIR}/system/hrpc 
+         ${HPCC_SOURCE_DIR}/system/mp 
+         ${HPCC_SOURCE_DIR}/system/include 
+         ${HPCC_SOURCE_DIR}/system/jlib 
+         ${HPCC_SOURCE_DIR}/system/jhtree
+         ${HPCC_SOURCE_DIR}/rtl/eclrtl
+         ${HPCC_SOURCE_DIR}/system/security/shared
+         ${HPCC_SOURCE_DIR}/system/security/securesocket
+         ${HPCC_SOURCE_DIR}/system/security/cryptohelper
+         ${HPCC_SOURCE_DIR}/testing/unittests
+         ${HPCC_SOURCE_DIR}/rtl/include
+         ${HPCC_SOURCE_DIR}/rtl/eclrtl
+         ${HPCC_SOURCE_DIR}/ecl/hql
+         ${HPCC_SOURCE_DIR}/common/deftype
+         ${HPCC_SOURCE_DIR}/dali/base
+         ${CMAKE_BINARY_DIR}
```

Comment:
@jakesmith since hql has been added to this project, to build correctly under LN we need to add ${CMAKE_BINARY_DIR}/oss here after the ${CMAKE_BINARY_DIR} entry.